ios/Video/RCTVideo.swift

```diff
@@ -883,7 +883,10 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
         _player?.pause()
         _player = nil
         _playerObserver.clearPlayer()
-
+        _imaAdsManager.getAdsManager()?.destroy()
```

Comment:
Are you really sure this is working ?
As _imaAdsManager is instanciated in init function, It will never be reallocated

won't   calling: _imaAdsManager.getAdsManager()?.pause() be better in that case ?

Reply:
No this is not working, i thought it might. I am not really an IOS developer, just tried it. 
I will try _imaAdsManager.getAdsManager()?.pause() and test and see the results. This issue is causing a delay in build and we really need this feature ASAP.

Reply:
@hassanrazaby433 any news ?